src/main/java/com/pontusvision/graphutils/gdpr.groovy

```diff
@@ -5624,12 +5624,15 @@ the end of the process.
 
   }
 
-  static def getDataRetentionKpis() {
+  static def getDataRetentionKpis(boolean test = false) {
 
     StringBuffer sb = new StringBuffer("[")
 
     boolean firstTime = true;
 
+//  today fixed for testing purpuses 1667569252000L = 2022-11-04T13:40:52Z [Friday, 4 November 2022 13:40:52]
+    def today = test?1667569252000L:"SYSDATE().asLong()"

```

Comment:
good approach, but don't add the string SYSDATE().asLong(); instead, get the num of ms since epoch directly.  I am not sure that what you have will actually work otherwise.